.azurepipelines/testcc.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       projects: '**/*.Tests.csproj'
       arguments: '--no-restore --framework ${{ parameters.framework }} --configuration ${{ parameters.configuration }} /p:CollectCoverage=true /p:Exclude="[*.Tests]*" /p:CoverletOutputFormat=cobertura'
   - script: |
-      bash <(curl -s https://codecov.io/bash)
+      bash <(curl -s https://codecov.io/bash) -v
```

Comment:
Is the verbose output intendent to be checked in?

Reply:
nope, artifact from debugging, fixed